### PR TITLE
Update copygb, copygb2 testing

### LINKED ISSUE #93: Handle change in ip library after 3.3.3
See #92 comments.

### PR DESCRIPTION
**Pull request by AlexanderRichert-NOAA**
Extend/reinstate copygb+copygb2 testing to include interp functionality. Move to ip 4.x.

Fixes #93
Addresses #116

### CODE REVIEW

**Comment by AlexanderRichert-NOAA** [comment]
wgrib output of 172.land.grib1 from copygb when using w3emc 2.9.3:
```
rec 1:0:date 2022122000 LAND kpds5=81 kpds6=1 kpds7=0 levels=(0,0) grid=172 sfc 24hr fcst:
  LAND=Land cover (land=1;sea=0) [fraction]
  timerange 10 P1 0 P2 24 TimeU 1  nx 690 ny 710 GDS grid 5 num_in_ave 0 missing 0
  center 7 subcenter 0 process 96 Table 2 scan: WE:SN winds(N/S) 
  polar stereo: Lat1 -36.866000 Long1 -220.194000 Orient -260.000000
     south pole (690 x 710) Dx 12700 Dy 12700 scan 64 mode 0
  min/max data 0 1  num bits 1  BDS_Ref 0  DecScale 0 BinScale 0
```
wgrib output of 172.land.grib1 from copygb when using w3emc 2.10.0:
```
rec 1:0:date 2022122000 LAND kpds5=81 kpds6=1 kpds7=0 levels=(0,0) grid=172 sfc 24hr fcst:
  LAND=Land cover (land=1;sea=0) [fraction]
  timerange 10 P1 0 P2 24 TimeU 1  nx 690 ny 710 GDS grid 5 num_in_ave 0 missing 0
  center 7 subcenter 0 process 96 Table 2 scan: WE:SN winds(N/S) 
  polar stereo: Lat1 -36.899000 Long1 -220.194000 Orient -80.000000
     south pole (690 x 710) Dx 12700 Dy 12700 scan 64 mode 0
  min/max data 0 1  num bits 1  BDS_Ref 0  DecScale 0 BinScale 0
```

**Comment by AlexanderRichert-NOAA** [comment]
@edwardhartnett per my above comment, when copygb regrids an input file, it gives different results for one of the test files depending on which version of w3emc is used. Note the differences for 'Lat1' and 'Orient'. Does this indicate that one or the other version of w3emc is doing something wrong, or does this come down to different valid choices that can be made in the regridding process?

**Comment by edwardhartnett** [comment]
@GeorgeGayno-NOAA do you know the answer to @AlexanderRichert-NOAA 's question? I sure don't...

**Comment by GeorgeGayno-NOAA** [comment]
> @edwardhartnett per my above comment, when copygb regrids an input file, it gives different results for one of the test files depending on which version of w3emc is used. Note the differences for 'Lat1' and 'Orient'. Does this indicate that one or the other version of w3emc is doing something wrong, or does this come down to different valid choices that can be made in the regridding process?

The definition of grids 172 and 220 was updated. https://github.com/NOAA-EMC/NCEPLIBS-w3emc/issues/157

**Comment by edwardhartnett** [comment]
OK, so it looks like we want to change the test to accept the answer from the current version of w3emc, and we can stop testing with older versions.

We should also change the grib_util main cmake file so that it requires the latest release of w3emc.

File: src/copygb/copygb.F90 [hunk 1]
```diff
@@ -1190,7 +1190,7 @@ END SUBROUTINE INTGRIB
 SUBROUTINE INTGRIB1(K1F,KGDS1F,K2F,KGDS2F,MRL,MRO, &
      IV,IP,IPOPT,KGDS1,K1,IB1,L1,F1,G1,KGDS2,K2, &
      IB2,L2,F2,G2,IRET)
-  USE IP_MOD
+!  USE IP_MOD
```
Comment (edwardhartnett) [inline src/copygb/copygb.F90]: What's going on here?

Comment (AlexanderRichert-NOAA) [reply src/copygb/copygb.F90]: I've been doing a lot of fiddling in investigating CI issues, so this is related to restesting with the older ip version. I'm currently trying to figure why the intel version is segfaulting.

Comment (edwardhartnett) [reply src/copygb/copygb.F90]: OK, sorry I didn't notice that this was a draft PR.

**Pull request closed without merging**